web/public/css/_variables.less

```diff
@@ -30,4 +30,4 @@
 @state-warning-bg: @cd-yellow;
 @state-warning-text: @cd-gray;
 @state-danger-bg: @cd-red;
-@state-danger-text: @cd-red;
+@state-danger-text: @cd-white;
```

Comment:
That's a bootstrap-wide change, do you see any scenario where we override the background on danger ?@DanielBrierton